pkg/planner/core/expression_rewriter.go

```diff
@@ -112,6 +122,20 @@ func buildExprWithAst(ctx expression.BuildContext, node ast.ExprNode, opts ...ex
 		return nil, errors.New("InputSchema and InputNames should be the same length")
 	}
 
+	if len(options.InputNames) > 0 {
+		intest.AssertFunc(func() bool {
```

Comment:
Is it possible to have the `if len(options.InputNames) > 0` inside the `intest.AssertFunc`?